proactive-healthcheck/api-health-check.sh

```diff
@@ -0,0 +1,574 @@
+#!/bin/bash
+#set -x
+
+#input parameters
+#username to connect to the ProActive Portals
+username=changeme
+#password of the user connecting to the ProActive Portals
+proactiveAdminPasswordDecrypted=changeme
+#server DNS name or public IP
+serverDNSName=try.activeeon.com
+#server Port
+webPort=8443
+#web protocol, http or https
+webProtocol=https
+#name of one of the node sources, needed in a test to get its description
+nsName=local
+
+#set colors for printf state of services
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+GREEN='\033[0;32m'
+PURPLE='\033[0;35m'
+
+#number of failed tests
+FAIL=0
+#number of successful tests
+SUCCESS=0
+
+#complete ProActive server web URL
+URL="${webProtocol}://${serverDNSName}:${webPort}"
+
+#function used by tests to check the returned status and print messages
+check_endpoint() {
+  endpoint=$1
+  message=$2
+  if [ "$endpoint" == "200" ]; then
```

Comment:
Are there any endpoints that return 201 or another successful HTTP code ?

Reply:
well, all of them are 200 but I added 201 as u have requested. It could be useful in the future. thx